### ISSUE TITLE
DebugMenu library

### ISSUE THREAD

**Issue by danielgomezrico** [artem-zinnatullin/qualitymatters#124]
Take a look to this repo https://github.com/palaima/DebugDrawer looks great

**Comment by artem-zinnatullin** [comment]
Yup, I know about it, but I'm ok with current solution, it's more custom and extensible from my perspective and it adds only 1 method to the apk.

If I'd do outsourcing development when you change project in 2-3 months, I'd definitely go with that library.

If I'd do long term in-house app development I'd go with custom solution which will fit needs more precisely.

@vanniktech what's your opinion on that?

**Comment by lenguyenthanh** [comment]
I agree with @artem-zinnatullin. Custom solution give us more power, flexible as well as testing. And it's not to difficult to develop.

**Comment by vanniktech** [comment]
I'm using this exact library in production (debug real package and release no-op) and so far I've only had positive encounters. In earlier versions there were some errors which are resolved now though. Today I found that one of the Module leaks but yeah .. :D

All in all I'm still really glad with the library. It provides some cool out of the box modules and also a way to create your own modules. (For instance I have one module for paperwork, one for backend related things etc.)

About testing, well I don't test that part of the app. Plus it's super easy to get a free navigation drawer with that library.

**Comment by artem-zinnatullin** [comment]
Great feedback!

Do you think that we should use it in qualitymatters?

>Today I found that one of the Module leaks but yeah .. :D

Hope you've opened an issue :)

**Comment by vanniktech** [comment]
I'd say we can use it here too. Though I actually never checked how you implemented the Drawer here in the app so I can't compare it. Though getting it for free is in my opinion better than implementing it yourself. 

Also the repository owner has accepted already a lot of my [PRs](https://github.com/palaima/DebugDrawer/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aclosed+author%3Avanniktech). He's still active and if we find something we can go and improve that library too. 

Plus the library has many artifacts on mavenCentral so we can pick what we need and don't need to include all 13 modules.

Yup I did https://github.com/palaima/DebugDrawer/issues/55

**Comment by caipivara** [comment]
I started another debug menu here https://github.com/BaristaVentures/debug-artist if you want to take a look

**Comment by artem-zinnatullin** [comment]
@caipivara yours looks nicer than DebugDrawer lib!
